frontend/src/pages/Player/RightPlayerPanel/components/NetworkResourcePanel/NetworkResourcePanel.tsx

```diff
@@ -248,11 +248,12 @@ function NetworkResourceDetails({
 	}, [resource.id])
 
 	useEffect(() => {
-		analytics.page(
-			`/sessions/${sessionSecureId}/network-resource/${
+		analytics.page('Network Resource Details', {
```

Comment:
as we discussed, does it make sense to update all the `.page()` calls with page names?

Reply:
Updated this and added `page: string` as a required argument on `analytics.page` to ensure we always do this.